### DISCUSSION TITLE
[feature request] results page for search

### DISCUSSION THREAD

**Discussion by jmbuhr** [quarto-dev/quarto-cli#3211] [Feature Requests]
I would love for quarto to have a page with the search results with it's own url so that query strings can be used, most notably for the quarto website itself.
e.g. `https://quarto.org/?search=test` to open the search results for `test`.

This would make it trivial to add a very helpful doc search functionality to the diverse quarto plugins, be it VS Code, RStudio or Neovim.

**Comment by dragonstyle** [comment]
I agree that a results page would be a very nice option. In the meantime, you can form urls to search results as such:

`https://quarto.org/?q=test&show-results=1`

which will show the results in the search overlay, if that is helpful...

**Reply by jmbuhr** [reply]
Yay, this is super helpful! Will add this to quarto-nvim straight away :)

**Reply by jmbuhr** [reply]
The feature is now live :)
https://github.com/quarto-dev/quarto-nvim

https://user-images.githubusercontent.com/17450586/200342098-906bb4fc-a855-4785-ae03-c138e8079940.mp4
